src/Proxy/Health/Node.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace OpenRuntimes\Proxy\Health;
+
+class Node
+{
+    private string $hostname;
+
+    private bool $online;
+
+    /**
+     * @var array<string,mixed>
+     */
+    private array $state;
+
+    /**
+     * @param array<string,mixed> $state
+     */
+    public function __construct(string $hostname, bool $online = false, array $state = [])
+    {
+        $this->hostname = $hostname;
+        $this->online = $online;
+        $this->state = $state;
+    }
+
+    public function getHostname(): string
+    {
+        return $this->hostname;
+    }
+
+    public function setOnline(bool $online): self
+    {
+        $this->online = $online;
+        return $this;
+    }
+
+    /**
+     * @param array<string,mixed> $state
```

Comment:
missing return type in doc blocks

Reply:
For consistency, Ill add proper doc blocks everywhere.